sample-apps/react/react-dogfood/hooks/useKeyboardShortcuts.ts

```diff
@@ -24,36 +24,41 @@ const isMacOS = () => {
 const [, raiseHandReaction] = defaultReactions;
 
 export const usePushToTalk = (key: string) => {
-  const [isTalking, setIsTalking] = useState(false);
-  const interactedRef = useRef(false);
-
   const { useMicrophoneState } = useCallStateHooks();
-  const { microphone } = useMicrophoneState();
+  const { microphone, isMute } = useMicrophoneState();
+  const hotkeyHandlerRef = useRef<(e: KeyboardEvent) => void>();
```

Comment:
Putting the event handler in a ref allows it it have access to the current microphone state, without the need to have microphone state as an effect dependency. This is kind of like what the [`useEvent` hook](https://github.com/reactjs/rfcs/blob/useevent/text/0000-useevent.md) does.